src/ui/session/content/message_row/reply.rs

```diff
@@ -97,19 +97,29 @@ impl MessageReply {
         let imp = self.imp();
 
         let message = self.message().unwrap();
-        let reply_to_message_id = message.reply_to_message_id();
         let is_outgoing = message.is_outgoing();
-        let chat = if message.reply_in_chat_id() != 0 {
-            message.chat_().session_().chat(message.reply_in_chat_id())
-        } else {
-            message.chat_()
+        let reply_to = message.reply_to().unwrap().0;
+
+        match reply_to {
+            tdlib::enums::MessageReplyTo::Message(reply_to) => {
+                let reply_to_message_id = reply_to.message_id;
+                let chat = if reply_to.chat_id != 0 {
+                    message.chat_().session_().chat(reply_to.chat_id)
+                } else {
+                    message.chat_()
+                };
+
+                if let Ok(message) = chat.fetch_message(reply_to_message_id).await {
+                    self.update_from_message(&message, is_outgoing);
+                } else {
+                    imp.message_label.set_label("Deleted message");
+                }
```

Comment:
Rather use a `match` statement. IMHO, it's much more readable than `if let else`